frontend/src/languages/en.json

```diff
@@ -20,7 +20,8 @@
     "login.msg.password.missing": "A password is required",
     "login.notice.message": "Notice: Access to this service is for authorized personnel only. If you do not have the expressed authorization of the administrator, you must exit now. This organization prohibits unauthorized access, disclosure, duplication, modification, diversion, destruction, loss, misuse, or theft of its information.",
     "error.invalidcredentials": "Username or Password are incorrect",
-    "patient.label.modify" : "Add/Modify Patient" ,
+    "patient.label.modify" :"Modify Patient" ,
```

Comment:
cc @mozzy11

Reply:
@sherrif10 ideally the component can either add a new patient or modify an existing one . 
That why the label was Add/Modify

Reply:
ohh Alright cool.

Reply:
but you need to change it both for the languages

Reply:
Resolved.